frontend/src/App.tsx

```diff
@@ -530,9 +471,24 @@ export class App extends PureComponent<Props, State> {
    * @param newReportProto a NewReport protobuf
    */
   handleNewReport = (newReportProto: NewReport): void => {
+    const initialize = newReportProto.initialize as Initialize
```

Comment:
Just out of curiosity, does the protobuf not return the correct type here?

Reply:
this is the larger protobufjs typescript issue: message variables are typed as interfaces, not classes (`IInitialize` instead of `Initialize`), and the interface typings claim that fields can be undefined - when in fact, for a deserialized protobuf, there's no such thing as an undefined field (because anything not explicitly defined gets assigned a default value). So the cast here prevents us from doing a whole lot of non-null assertions on the message fields.